appinventor/docs/reference/other/extensions.html

```diff
@@ -414,7 +414,7 @@
       <header class="clearfix" role="banner">
         <div id="top_bar">
           <div id="branding">
-            <div id="logo"><a href="http://appinventor.mit.edu" title="Home page" class="active"><img src="/reference/components/images/logo2.png" alt="Explore MIT App Inventor logo"></a></div>
+            <div id="logo"><a href="http://appinventor.mit.edu" title="Home page" class="active"><img src="/reference/components/images/codi_long.png" alt="Explore MIT App Inventor logo"></a></div>
```

Comment:
In the other files you modified the width/height of the logo image, but not in this one. You should update the CSS rules to be consistent.

Reply:
Oops, well spotted!